### PR TITLE
Create Lmcraft --> Image uploadeing Soon

### PR DESCRIPTION
**Pull request by Luispupuis**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by Luispupuis** [comment]
How can I edit this ?

File: minecraft_servers/Lmcraft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "Lmcraft",
+  "nice_name": "Lmcraft",
+  "direct_ip": "Lmcraft.de",
+  "server_wildcards": [
+    "%.Lmcraft.de"
+  ],
+  "social": {
+    "web": "https://Lmcraft.de",
```
Comment (maaaathis) [inline minecraft_servers/Lmcraft/manifest.json:9]: Not reachable

File: minecraft_servers/Lmcraft/manifest.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "Lmcraft",
+  "nice_name": "Lmcraft",
+  "direct_ip": "Lmcraft.de",
+  "server_wildcards": [
+    "%.Lmcraft.de"
```
Comment (maaaathis) [inline minecraft_servers/Lmcraft/manifest.json:6]: Doesn't look like a real wildcard
image

File: minecraft_servers/Lmcraft/manifest.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "Lmcraft",
```
Comment (maaaathis) [inline minecraft_servers/Lmcraft/manifest.json:2]: `server_name` needs to be lowercased (you need also to rename your created directory)
```suggestion
  "server_name": "lmcraft",
```

**Review by maaaathis — CHANGES_REQUESTED** [review]
You need to upload at least a `icon.png` and a `icon@2x.png`. For reference see here: https://github.com/LabyMod/server-media/blob/master/docs/Files.md
Also your `server_name` and so also your directory needs to be lowercased. See my comment below.

**Pull request closed without merging**